frontend/src/lib/components/apps/editor/settingsPanel/inputEditor/JsonEditor.svelte

```diff
@@ -1,12 +1,33 @@
 <script lang="ts">
 	import SimpleEditor from '$lib/components/SimpleEditor.svelte'
+	import { deepEqual } from 'fast-equals'
 	export let code: string | undefined
 	export let value: any = undefined
 	export let error = ''
 	export let editor: SimpleEditor | undefined = undefined
 	export let small = false
 
 	$: tooBig = code && code?.length > 1000000
+
+	function setCodeWhenValueChanges() {
```

Comment:
When the prop value changes, the code prop becomes out of sync, so I manually set the code in the editor in that case

Reply:
Seems not related to this PR and also it's better to have an explicit call to  a JsonEditor function if props can change while the editor is open.